docs/ftldns/configfile.md

```diff
@@ -56,6 +56,32 @@ How should `FTL` reply to blocked queries?<br>
 #### `DBFILE=/etc/pihole/pihole-FTL.db` {#dbfile data-toc-label='DB File'}
   Specify path and filename of FTL's SQLite3 long-term database. Setting this to `DBFILE=` disables the database altogether
 
+### File options
+
+#### `LOGFILE=/var/log/pihole-FTL.log` {#file_LOGFILE data-toc-label='Log file'}
+  Location of FTL's log file. If you want to move the log file to a different place, also consider [this FAQ article](https://discourse.pi-hole.net/t/moving-the-pi-hole-log-to-another-location-device/2041).
```

Comment:
We should probably only recommend to use the config file, as the symlink could cause issues as the article notes. What benefits does the symlink have that the config file doesn't?

Reply:
The log file location is hardcoded, e.g., in the debugger. With a symlink, it could still be found.

Reply:
The debugger should then be updated to use the config file for the location. Otherwise, there is no point in supporting the config option if it isn't supported by the rest of the software.

Reply:
This is true, however, it is out of scope for the current PR that just wants to bring the documentation to the current state of what we released in `master`.

Reply:
See https://github.com/pi-hole/pi-hole/pull/2837